packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -12,7 +12,7 @@ import { Classes as CoreClasses, IProps, Popover, Position } from "@blueprintjs/
 
 import * as Classes from "../common/classes";
 import { LoadableContent } from "../common/loadableContent";
-import { HeaderCell, IHeaderCellProps } from "./headerCell";
+import { HeaderCell, IHeaderCellProps, IInternalHeaderCellProps } from "./headerCell";
```

Comment:
@giladgray reflecting back on our discussion yesterday about `Select`/`Suggest` naming, here are the two props interface names in this context. Don't love them as is, happy to change to "core"/"base" terminology, e.g.